support/src/main/resources/org/jenkinsci/plugins/workflow/support/visualization/table/FlowGraphTable/ajax.jelly

```diff
@@ -41,9 +41,11 @@
 
         <j:forEach var="row" items="${it.rows}">
           <j:set var="node" value="${row.node}"/>
-          <tr>
+          <tr style="opacity: ${row.isExecuted()?'1.0':'0.3'}"> <!-- Muted if not executed -->
             <!-- tooltip is for now debugging only -->
-            <td style="padding-left: ${row.treeDepth*20+5}px" tooltip="ID: ${node.id}">
+            <j:set var="exec_state" value=""/>
+            <j:if test="${!row.executed}"><j:set var="exec_state" value=" (${%not executed})"/></j:if>
+            <td style="padding-left: ${row.treeDepth*20+5}px" tooltip="ID: ${node.id}${exec_state}">
```

Comment:
`exec_state` came about because I was unable to figure out how to nest `${}` expressions inside each other. In any case, this is perfectly fine too.